test/udp_sender.js

```diff
@@ -94,12 +95,13 @@ describe('udp sender should', () => {
     });
 
     it ('span references serialize', (done) => {
-        let context = tracer.startSpan('just-used-for-context').context();
-        let context2 = tracer.startSpan('just-used-for-context').context();
+        let childOfContext = tracer.startSpan('just-used-for-context').context();
+        let followsFromContext = tracer.startSpan('just-used-for-context').context();
+        followsFromContext.traceId = childOfContext.traceId;
         let spanOne = tracer.startSpan('operation-one', {
             references: [
-                new opentracing.Reference(opentracing.REFERENCE_CHILD_OF, context),
-                new opentracing.Reference(opentracing.REFERENCE_FOLLOWS_FROM, context2)
+                new opentracing.Reference(opentracing.REFERENCE_CHILD_OF, childOfContext),
+                new opentracing.Reference(opentracing.REFERENCE_FOLLOWS_FROM, followsFromContext)
```

Comment:
I suggest a table-driven test for various permutations, e.g.
 * childOf=null, refs=[]
 * childOf + refs=[]
 * childOf + refs=[follows]
 * childOf + refs=[childOf, follows]
 * childOf=null + refs=[childOf]
 * childOf=null + refs=[follows]  // this should still inherit parentID from the follows ref
 * childOf=null + refs=[follows, childOf]